platalea/basic.py

```diff
@@ -84,10 +85,8 @@ def val_loss(net):
 
     if not wandb_log:
         wandb_log = config
-
-    logging.getLogger().info(
-        "Run 'wandb disabled' if you don't want to use wandb cloud logging.")
```

Comment:
Why was this removed? I thought we decided that it would help users finding their way without having to read through the whole documentation. Did the warning go somewhere else perhaps?

Reply:
I removed it as wandb is disabled by default in the example we provide. I just added docstring with an explanation of `wandb_mode` parameter. Do you think that's enough or should we also provide that information elsewhere?

Reply:
In that case it makes sense. Good change :-)